[Az.RecoveryServices.Backup] Workload Recovery config, disallowing target item to be an availability group (#13851)
Azure/azure-powershell: src/RecoveryServices/RecoveryServices/ChangeLog.md

        - Additional information about change #1
-->
## Upcoming Release

> * Workload config, blocking target item to be an availability group.

Reviewer: How about
```suggestion
* Blocked getting workload config when target item is an availability group.
```
